backend/src/main/kotlin/app/ehrenamtskarte/backend/auth/database/Setup.kt

```diff
@@ -1,13 +1,9 @@
 package app.ehrenamtskarte.backend.auth.database
 
-import app.ehrenamtskarte.backend.auth.database.repos.AdministratorsRepository
 import org.jetbrains.exposed.sql.SchemaUtils
 
 fun setupDatabase() {
     SchemaUtils.create(
         Administrators
     )
-    if (AdministratorEntity.count() == 0L) {
-        AdministratorsRepository.insert("admin@example.com", "admin")
-    }
```

Comment:
I silently want to remove this.

Reply:
I will take care that we have some login on production and staging.

Reply:
We should have some easy and simple way to initialize local setups as well

Reply:
I created a task: https://github.com/ehrenamtskarte/ehrenamtskarte/issues/410